definitions/DisplaySeason.dbd

```diff
@@ -5,13 +5,7 @@ int Field_9_2_0_41827_001? // GetCurrentSeasonValues.rewardSeasonID?
 int ExpansionID?
 locstring Field_9_2_0_42174_000_lang? // WeeklyRewardActivityInfo.raidString?
 int Field_10_0_2_45779_004?
-locstring Field_11_0_0_55000_000_lang?
-int Field_11_0_0_55000_001?
-int Field_11_0_0_55000_002?
-int Field_11_0_0_55000_003?
-int Field_11_0_0_55000_004?
-int Field_11_0_0_55000_005?
-int Field_11_0_0_55000_006?
+int Field_11_0_0_55000_000?
```

Comment:
fyi: established practice is to keep the autogenerated name, i.e. `Field_11_0_0_55000_006` in this case, making it clear this is column 006 in that build. But that's fine, I guess.